mrp_unbuild_cust_alu_analytics/views/mrp_bom_views.xml

```diff
@@ -0,0 +1,27 @@
+<odoo>
+    <data>
+        <!-- Form View -->
+        <record id="mrp_bom_form_process_type_form_view" model="ir.ui.view">
+            <field name="name">mrp.bom.view.form (process_id)</field>
+            <field name="model">mrp.bom</field>
+            <field name="inherit_id" ref="mrp.mrp_bom_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='product_tmpl_id']" position="before">
+                    <field name="process_type_id"/>
+                </xpath>
+            </field>
+        </record>
+
+        <!-- Tree View -->
+        <record id="mrp_bom_form_process_type_tree_view" model="ir.ui.view">
+            <field name="name">mrp.bom.view.tree (process_id)</field>
+            <field name="model">mrp.bom</field>
+            <field name="inherit_id" ref="mrp.mrp_bom_tree_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='type']" position="before">
+                    <field name="process_type_id" optional="0"/>
```

Comment:
Optional, Show or Hide?

Reply:
Hide

Reply:
OK, then let's use `hide` instead of `0`.

Reply:
Done